nexus/tests/integration_tests/instances.rs

```diff
@@ -123,6 +123,65 @@ async fn test_instances_access_before_create_returns_not_found(
     );
 }
 
+#[nexus_test]
```

Comment:
I'm assuming this isn't as exhaustive a test as what we had for the old APIs.  Is there a plan for converting them?

Reply:
Yes, absolutely.